### ISSUE TITLE
Grunt file to publish to Atmosphere

### ISSUE THREAD

**Issue by dandv** [MeteorCommunity/discussions#15]
Grunt patch has its own repo - https://github.com/MeteorCommunity/grunt-meteor

**Comment by raix** [comment]
Maybe have grunt create bower/jspm/jam/browserify/npm configs from a single meteor package.js?
We could create a small tool that would generate the bower.json/component.json/package.json files and add the proper require/amd/umd code to the library code.

eg.:

``` bash
/lib                   # the source code
/dist                  # the generated distribution files
/meteor/package.js     # The meteor config
/bower.json            # The generated config for bower
/components.json       # The generated config for components
/package.json          # The generated config for npm/browserify/jam/jspm etc.
/README.md             # The Meteor compliant readme
```

I think that it would help library writers support all package systems and make cleaner packges for Meteor. It could be via grunt or a custom tool.

EDIT: Would be nice to skip the requirejs/global exports etc. in the Meteor version of the package/library

**Comment by zimme** [comment]
gulpfile.js could be useful too for repos using gulp instead of grunt =)

**Comment by zimme** [comment]
Would it be possible to create the package.js from package.json/bower.json if they are available, inside the tasks?

**Comment by raix** [comment]
Yes, i may create a node package with this, Can be used more places https://github.com/raix/Meteor-include/issues/2

**Comment by dandv** [comment]
@splendido thanks for the inspiration! I've posted a Gruntfile diff in the OP. We have a minor problem now: https://github.com/MeteorCommunity/grunt-meteor/issues/1

**Comment by raix** [comment]
Would be nice I think if the complete work flow is rigged for them - eg. help them create the user account/login/bump making it really easy for them - it could event generate the package in `.meteor` - have this added to .gitignore would keep their repo clean.

**Comment by raix** [comment]
@dandv maybe publish grunt/gulp integrations at something like MeteorCommunity/grunt-publish etc.?

**Comment by dandv** [comment]
@raix - done, I'll let you update this issue.
